bridge/server/services/api-service.ts

```diff
@@ -104,4 +104,12 @@ export class ApiService {
   public getShipyard(projectName: string): Promise<AxiosResponse<Resource>> {
     return this.axios.get<Resource>(`${this.baseUrl}/configuration-service/v1/project/${projectName}/resource/shipyard.yaml`);
   }
+
+  public getServiceResource(projectName: string, stageName: string, serviceName: string, nextPageKey?: number): Promise<AxiosResponse<ResourceResponse>> {
+    let url = `${this.baseUrl}/configuration-service/v1/project/${projectName}/stage/${stageName}/service/${serviceName}/resource`;
+    if (nextPageKey) {
+      url += `?nextPageKey=${nextPageKey}`;
+    }
+    return this.axios.get<ResourceResponse>(url);
```

Comment:
I don't really like the approach to add query parameters this way because it could easily raise some problems (caused by invalid provided characters like space or percent). The way how to add query parameters is:
```ts
const url = `${this.baseUrl}/configuration-service/v1/project/${projectName}/stage/${stageName}/service/${serviceName}/resource`;
const params: {[key: string]: string} = {};
if (nextPageKey) {
  prams.nextPageKey = String(nextPageKey);
}
return this.axios.get<ResourceResponse>(url, params);
```